src/sentry.ts

```diff
@@ -21,17 +23,20 @@ const PINO_TO_SENTRY: { [key: number]: Severity } = {
     60: Severity.Critical,
 };
 
-class SentryTransformStream extends Transform {
-    // tslint:disable-next-line:function-name
-    public _transform(chunk: any, _encoding: string, callback: TransformCallback) {
-        const obj = JSON.parse(chunk);
-        withScope(scope => {
-            scope.setLevel(PINO_TO_SENTRY[obj.level]);
-            scope.setExtras(obj);
-            reportToSentry(obj);
-        });
-        this.push(chunk);
-        callback();
-    }
-}
-export { SentryTransformStream };
+export const createSentryTransformStream = (options: CosmasOptions): any => {
+    return class SentryTransformStream extends Transform {
+        // tslint:disable-next-line:function-name
+        public _transform(chunk: any, _encoding: string, callback: TransformCallback) {
+            const obj = JSON.parse(chunk);
+            if ((options.sentryLevel || levels.warn) <= obj.level) {
```

Comment:
Would you still like it with operands switched? :) 
```suggestion
            if (obj.level >= (options.sentryLevel || levels.warn)) {
```

Reply:
you know I would :heart: